### PR TITLE
Update K8s "Hello World" - Node.js

### PR DESCRIPTION
**Pull request by kelsk**

### CODE REVIEW

File: nodejs/nodejs-hello-world/src/index.html.hbs [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html lang=en>
+<head>
+<meta charset=utf-8>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>Congratulations | Kubernetes</title>
```
Comment (simonz130) [inline nodejs/nodejs-hello-world/src/index.html.hbs:6]: Congratulations on deploying your app on Kubernetes

File: nodejs/nodejs-hello-world/src/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const express = require('express');
```
Comment (simonz130) [inline nodejs/nodejs-hello-world/src/index.js:1]: Likely you just need to remove the app.js that is now replaced with index.js

File: nodejs/nodejs-hello-world/src/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+const {readFileSync} = require('fs');
```
Comment (simonz130) [inline nodejs/nodejs-hello-world/src/index.js:2]: Add copyright

File: nodejs/nodejs-hello-world/src/index.html.hbs [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="preload" as="font">
+<link
+  href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAACXBIWXMAAAsSAAALEgHS3X78AAAB5klEQVRYha2XMVLDMBBFPxl6qCjciBskNzA3AE5AqrTQuaVVR1pX4QSYGyQ3SGqGATcu3JBUlGGUiTPyeiWvZH6TsSTLb/+XN8nZfr+H0lUBYAvgucySbzBSupoCmJRZ8sTNx2p0vG8J4AHAl9LVQunqmtnPQD4qXa0d81GyARo1IC/2hmWWGIfeAYwBGIi7fwMos2QNYEfmTLWXZKw4fl4AeKOQQxwAcaERrbIg14MjCQKwYrA1KJI+gBtmjLqAIZEcXsNGSlfb42a27sssKaw15lz8ePbcGOdcrzPViFz3uuCIwVZQJBIAbiMuBlviSCQASulqEgjQqPctaQE4+gEiYrDljYQ6AIcLU2ZM6gJ8kUgBxp6uGKJOJFIACJuSRK1IOgCecxDzNrh0iuScW/D7+eG4LxGuk6sDcDVbTZhuCEe1sV/JxuFpnacF5wDX/zsAV7OVOZS3EQ8/tOo6Tw+tWgqwqfN0S8Ziqp/Xedr6SScFWDBjIQAny+lEC8CT/5KsC7G/ZbkXwFF9WefpmoxJq+9YHgMQc/qdlscAhNrvtdwJ4Mh/x1Thq77XcieApHoPgNjyUABJ8wmyfBAAU32w5SyAI/+5p/tFW84CkOpfzb9kaqll/yDLfQDsgy2Z6gdb3hKAP2n+6m+fxBVHAAAAAElFTkSuQmCC"
+  rel="icon" 
+  type="image/png"
```
Comment (simonz130) [inline nodejs/nodejs-hello-world/src/index.html.hbs:11]: This is cloud run icon. You probably want kubernetes or remove this altogether

Comment (simonz130) [reply nodejs/nodejs-hello-world/src/index.html.hbs:11]: We might need a different logo in the pony above where it says "it's running". Currently it's a cloud run logo

Comment (kelsk) [reply nodejs/nodejs-hello-world/src/index.html.hbs]: I should have an updated Kubernetes-specific unicorn logo merged into this code by 09/25/20.

File: nodejs/.ci/nodejs-hello-world.cloudbuild.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    waitFor: ['Integration tests']
+
+options:
+    env:
+    - CLOUDSDK_COMPUTE_ZONE=us-central1-a
+    - CLOUDSDK_CONTAINER_CLUSTER=staging
+    - SKAFFOLD_DEFAULT_REPO=gcr.io/$PROJECT_ID
+    - TEST_NAMESPACE=test-$BUILD_ID-nodejs-hello-world
+    - HELLO_WORLD_SERVICE=nodejs-hello-world-external
+    - GUESTBOOK_SERVICE=nodejs-hello-world-frontend
```
Comment (simonz130) [inline nodejs/.ci/nodejs-hello-world.cloudbuild.yaml:76]: You don't need this variable here

Comment (simonz130) [reply nodejs/.ci/nodejs-hello-world.cloudbuild.yaml:76]: or at least the name doesn't make sense

Comment (kelsk) [reply nodejs/.ci/nodejs-hello-world.cloudbuild.yaml]: Removed (that var was leftover from copying code from the Guestbook test)

**Pull request closed without merging**